webui/react/src/services/types.ts

```diff
@@ -98,6 +98,7 @@ export interface GetExperimentsParams extends PaginationParams {
 
 export interface SearchExperimentsParams extends PaginationParams {
   projectId?: number;
+  sort?: string;
   /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
   options?: any;
```

Comment:
I know you didn't add it in this PR, but why is `options` in this type?

Reply:
My guess is that whoever created this type got confused with the options parameter that can go in the api request section. I think you're free to remove it.